web/src/js/ducks/modes/regular.ts

```diff
@@ -18,8 +18,11 @@ export const initialState: RegularState = {
 };
 
 export const getMode = (modes) => {
-    const regularMode = modes.regular;
-    return includeModeState("regular", regularMode);
+    const regularMode: RegularState = modes.regular;
+    if (!regularMode.error) {
```

Comment:
Can't we have `includeModeState` handle this? :)

Reply:
sure :) let me change it